docs/advanced/debugging.rst

```diff
@@ -71,6 +71,7 @@ The quick start above shows how you can get going with the ``Autofac.Diagnostics
 When the ``OperationCompleted`` event gets raised, you'll get event arguments that provide:
 
 * ``Operation`` - the actual resolve operation that completed so you can inspect it if needed.
+* ``OperationSucceeded` - a boolean value indicating whether the contained trace is for a successful or failed operation.
```

Comment:
```suggestion
* ``OperationSucceeded` - a Boolean value indicating whether the contained trace is for a successful or failed operation.
```

I'm somewhat famous on my team at work for being pedantic about `Boolean`. There was a code review once that I still get a lot of flack for because I explained how it's [a proper noun based on George Boole](https://en.wikipedia.org/wiki/Boolean_data_type) and now I can't live it down. So... I may as well embrace it. 😆

Reply:
Oh, and I just found a missing backtick, so... less pedantic?

Reply:
Haha, I will have to remember that. So bool 👍 and Boolean 👍?

Reply:
Yeah, `bool`, I think, is more about C# standards where `Boolean`, the full name, is based on the dude.